lib/takeaway.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'twilio'
+class Takeaway
+
+  attr_reader :menu, :orders, :message_sender, :total_order_value
+
+  def initialize(message_sender = TwilioSender.new)
+    @menu = {:FishAndChips => 5, :MeatAndTwoVeg => 7}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.